### ISSUE TITLE
"Version mismatch error.  This is libtool 2.4.6 Debian-2.4.6-0.1, but the definition of this LT_INIT comes from an older release."

### ISSUE THREAD

**Issue by flannelinsummer** [onkelDead/tascam.lv2#3]
Hello @onkelDead  ,

First of all, thank you for your kind efforts to provide further support for the TASCAM US-16x08 to Linux users when TASCAM has not done so.

I am an Ardour User (Version 5.12.0) and an owner of the TASCAM US-16x08.

The Linux kernel version I am using is 4.15.0-91-lowlatency

I am trying to use the tascam.lv2 plugins within Ardour, but I am having some issues, unfortunately. I hope that you may be able to help me. 

First I have cloned the git project into a folder on my computer:
`git clone https://github.com/onkelDead/tascam.lv2.git`

I thought that using the LV2 plugins you have created may have been as simple as copying the "tascam.lv2" directory (with all files included) into my computer's "usr/lib/lv2" directory, and then running the plugin search within Ardour. But when I did so, the plugins are not found when I try to add them to a channel in Ardour.

So then, back in where I cloned the git project, I have changed the permissions of the "ltmain.sh" file found in "/tascam.lv2" to "Allow executing file as program".

Then in a terminal in the folder that contains ltmain.sh, I tried running `sh ltmain.sh`, but the following error is displayed:

```
ltmain.sh: Version mismatch error.  This is libtool 2.4.6 Debian-2.4.6-0.1, but the
ltmain.sh: definition of this LT_INIT comes from an older release.
ltmain.sh: You should recreate aclocal.m4 with macros from libtool 2.4.6 Debian-2.4.6-0.1
ltmain.sh: and run autoconf again.
```
I feel that there may be a very simple solution to the error above, but I am having trouble finding it. Are you able to provide a suggestion?

Any help will be greatly appreciated and I would be happy to donate to your project if there is a way for me to do that.

**Comment by onkelDead** [comment]
Hi,

the fact, that the script _libtool_ is contained in repository is more or less a fault. It will be recreated on your machine during build commands.

To be complete:

- re-clone the repository
- change in its directory
- execute the following sequence on commands:

1. aclocal
2. automake
3. ./configure
4. make
5. sudo make install 

Apart from your inspection, the _make install_ will copy then build results and ttl-files to /_usr/local/lib/lv2/tascam.lv2_. Ardour will search for plugins there also. To change to common _/usr/lib/lv2_, you have to add a prefix option to _./configure_ . (See ./configure --help)

Greets,
onkel

**Comment by flannelinsummer** [comment]
Thank you. I was able to install this package by following the steps you described. The plugin files were installed in directory /usr/local/lib/lv2/tascam.lv2 and are were found by Ardour during its normal startup plugin scan and I am able to control the plugins in Ardour.

I confirm that the Tascam Equalizer plugin is now working. Confirmation was made by listening to the sound being processed by the effect and by viewing the frequency response of the channel following the equalizer plugin. 
However, the compressor plugins is not affecting the audio signal from the Tascam US1x08 unit. I have selected the channel to control. I hear no influence of the plugin and there is also no movement of the "Compression Level" visual indicator within the plugin even when using extreme settings to maximize compression. 

I believe I may need to install your "tascam-driver-patch" before I will be able to control the built in compressor(s) of the Tascam US16x08.

Best regards,
Justin

**Comment by onkelDead** [comment]
BTW. I digged out, that both (tascam-gtk & tascam.lv2) wont work together. May be this is what you've detected.

**Comment by flannelinsummer** [comment]
I think that is not the reason that the compressor plugin in tascam.lv2 was not working for me because I don't believe I ever attempted to run tascam.lv2 and tascam-gtk simultaneously. The compressor plugin (tascam.lv2) not having an effect on the input signal when using it it in Ardour is was what prompted me to next install tascam-gtk to see if that would cause the compressor plugin to work, and that's when I found that the tascam-gtk _did_ effectively control the compressor plugin. If I recall correctly,  I was not running tascam-gtk and tascam.lv2 simultaneously and I'm sure that I was not when I first tested the tascam.lv2 compressor and it was not fully working beause I had not yet installed tascam-gtk.

Overall, being able to use Tascam-GTK to control the unit and being able to save presets within Tascam-GTK is enough for me and I am grateful for your release!
